web.rb

```diff
@@ -104,7 +104,7 @@ def clear_bot_cache(bot)
 end
 
 def bot_relay(bot, message)
-  return 'osusumeなら俺の隣で寝てるよ？' if bot == 'osusume'
+  return 'osusumeなら俺の隣で寝てるよ？' if bot == 'osusume' && message.empty?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.